src/cbmc/bmc.cpp

```diff
@@ -78,15 +78,16 @@ void bmct::error_trace()
   switch(ui)
   {
   case ui_message_handlert::uit::PLAIN:
-    std::cout << "\n" << "Counterexample:" << "\n";
-    show_goto_trace(std::cout, ns, goto_trace);
+    status() << "Counterexample:" << eom;
+    show_goto_trace(status(), ns, goto_trace);
+    status() << eom;
     break;
 
   case ui_message_handlert::uit::XML_UI:
     {
       xmlt xml;
       convert(ns, goto_trace, xml);
-      std::cout << xml << "\n";
+      status() << xml << eom;
```

Comment:
@smowton, although this is fine for PLAIN, for JSON and XML this prints the trace into a STATUS-MESSAGE, which is not intended. Please revert. We can only do this after having refactored the message_streamt to support piping out structured objects.